include/odp/api/spec/ipsec.h

```diff
@@ -1223,9 +1223,12 @@ typedef struct odp_ipsec_status_t {
  * Each successfully transformed packet has a valid value for these metadata
  * regardless of the inner packet parse configuration
  * (odp_ipsec_inbound_config_t):
- * - L3 offset: Offset to the first byte of the (outmost) IP header
- * - pktio:     For inline IPSEC processed packets, original packet input
- *              interface
+ * - L3 offset:       Offset to the first byte of the (outmost) IP header
+ * - IPv4/IPv6 flags: These flags can be used to determine packet contents.
+ *                    If neither IPv4 nor IPv6 flag are set, received packet is
+ *                    TFC dummy packet and should be dropped by an application
+ * - pktio:           For inline IPSEC processed packets, original packet input
+ *                    interface
```

Comment:
Is this not equally applicable to `odp_ipsec_in_enq()`? And what about packets received inline? Are we assuming that TFC dummy packets are dropped by the inline RX path? That should also be documented if yes. If not then application must be prepared to receive them that way as well.

Reply:
`odp_ipsec_in_enq()` just refers to `odp_ipsec_in()` function. As for inline processing, an application should be prepared to receive them, because some hardware can not drop them.